src/VisualStudio/Core/Def/Implementation/ProjectSystem/Legacy/AbstractLegacyProject_IVsENCRebuildableProjectCfg.cs

```diff
@@ -18,69 +19,44 @@ public int HasCustomMetadataEmitter(out bool value)
         }
 
         public int StartDebuggingPE()
-        {
-            return _editAndContinueProject.StartDebuggingPE();
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int StopDebuggingPE()
-        {
-            return _editAndContinueProject.StopDebuggingPE();
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int GetPEidentity(Guid[] pMVID, string[] pbstrPEName)
-        {
-            return _editAndContinueProject.GetPEidentity(pMVID, pbstrPEName);
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int EnterBreakStateOnPE(EncInterop.ENC_BREAKSTATE_REASON encBreakReason, ShellInterop.ENC_ACTIVE_STATEMENT[] pActiveStatements, uint cActiveStatements)
-        {
-            return _editAndContinueProject.EnterBreakStateOnPE(encBreakReason, pActiveStatements, cActiveStatements);
-        }
+            => throw ExceptionUtilities.Unreachable;
+
+        public int ExitBreakStateOnPE()
+            => throw ExceptionUtilities.Unreachable;
 
         public int GetExceptionSpanCount(out uint pcExceptionSpan)
-        {
-            pcExceptionSpan = default;
-            return _editAndContinueProject.GetExceptionSpanCount(out pcExceptionSpan);
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int GetExceptionSpans(uint celt, ShellInterop.ENC_EXCEPTION_SPAN[] rgelt, ref uint pceltFetched)
-        {
-            return _editAndContinueProject.GetExceptionSpans(celt, rgelt, ref pceltFetched);
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int GetCurrentExceptionSpanPosition(uint id, VsTextSpan[] ptsNewPosition)
-        {
-            return _editAndContinueProject.GetCurrentExceptionSpanPosition(id, ptsNewPosition);
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int GetENCBuildState(ShellInterop.ENC_BUILD_STATE[] pENCBuildState)
-        {
-            return _editAndContinueProject.GetENCBuildState(pENCBuildState);
-        }
-
-        public int ExitBreakStateOnPE()
-        {
-            return _editAndContinueProject.ExitBreakStateOnPE();
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int GetCurrentActiveStatementPosition(uint id, VsTextSpan[] ptsNewPosition)
-        {
-            return _editAndContinueProject.GetCurrentActiveStatementPosition(id, ptsNewPosition);
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int EncApplySucceeded(int hrApplyResult)
-        {
-            return _editAndContinueProject.EncApplySucceeded(hrApplyResult);
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int GetPEBuildTimeStamp(Microsoft.VisualStudio.OLE.Interop.FILETIME[] pTimeStamp)
         {
             return VSConstants.E_NOTIMPL;
```

Comment:
💡 Can you document the reason why the last two methods in this type are different (`GetPEBuildTimeStamp` and `HasCustomMetadataEmitter`)

Reply:
Change them to throw as well.